programs/marginfi/src/state/fee_state.rs

```diff
@@ -23,11 +23,14 @@ pub struct FeeState {
     /// Flat fee assessed when a new bank is initialized, in lamports.
     /// * In SOL, in native decimals.
     pub bank_init_flat_sol_fee: u32,
+    /// Flat fee assessed at flashloan end, in lamports.
```

Comment:
This struct is live on mainnet now, so new fields need to go at the end (just before padding) to avoid breaking existing serialization.

Reply:
done, let me know if it's preferred that I move the `flashloan_flat_sol_fee`to be the last arg in all the methods as well. I had put it next to the other "flat_sol" fee arbitrarily.